`aarch64-unknown-openbsd` fails to build due to missing `parse_system_registers`

<!--
Thank you for filing a bug report! 🐛 Please provide a short summary of the bug,
along with any information you feel relevant to replicating the bug.
-->

I tried to build something using `--target aarch64-unknown-openbsd`.

I expected to see this happen: Stdlib would compile successfully.

Instead, this happened: 

```text
  error[E0425]: cannot find function `parse_system_registers` in module `super::aarch64`
    --> /home/runner/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std_detect/src/detect/os/openbsd/aarch64.rs:34:21
     |
  34 |     super::aarch64::parse_system_registers(aa64isar0, aa64isar1, aa64mmfr2, aa64pfr0)
     |                     ^^^^^^^^^^^^^^^^^^^^^^ not found in `super::aarch64`
     |
  help: consider importing this function
     |
   7 + use crate::detect::aarch64::parse_system_registers;
     |
  help: if you import `parse_system_registers`, refer to it directly
     |
  34 -     super::aarch64::parse_system_registers(aa64isar0, aa64isar1, aa64mmfr2, aa64pfr0)
  34 +     parse_system_registers(aa64isar0, aa64isar1, aa64mmfr2, aa64pfr0)
```

### Meta
<!--
If you're using the stable version of the compiler, you should also check if the
bug also exists in the beta or nightly versions.
-->

`rustc --version --verbose`:
```
rustc 1.93.0-nightly (01867557c 2025-11-12)
binary: rustc
commit-hash: 01867557cd7dbe256a031a7b8e28d05daecd75ab
commit-date: 2025-11-12
host: x86_64-unknown-linux-gnu
release: 1.93.0-nightly
LLVM version: 21.1.5
```

--- a/library/std_detect/src/detect/os/openbsd/aarch64.rs
+++ b/library/std_detect/src/detect/os/openbsd/aarch64.rs
@@ -31,7 +31,7 @@ pub(crate) fn detect_features() -> cache::Initializer {
     // the feature is available.
     let aa64pfr0 = sysctl64(&[libc::CTL_MACHDEP, CPU_ID_AA64PFR0]);
 
-    super::aarch64::parse_system_registers(aa64isar0, aa64isar1, aa64mmfr2, aa64pfr0)
+    crate::detect::aarch64::parse_system_registers(aa64isar0, aa64isar1, aa64mmfr2, aa64pfr0)
 }
 
 #[inline]

